cooperative_lanechange/src/cooperative_lanechange.cpp

```diff
@@ -260,6 +293,10 @@ namespace cooperative_lanechange
 
         //if ack mobility response, send lanechange response
         if(!negotiate || is_lanechange_accepted_){
+            ROS_DEBUG_STREAM("negotiate" << negotiate);
+            ROS_DEBUG_STREAM("negotis_lanechange_accepted_iate" << is_lanechange_accepted_);
```

Comment:
Looks like typos in the log message, let's clean some of those up a bit before release.